docs/src/adoc/index.adoc

```diff
@@ -1628,6 +1628,16 @@ As in the Core API, neither `@BindMap`, `@BindBean`, `@BindFields`, nor
 `@BindMethods` support binding of nested properties
 (e.g. `:user.address.street`).
 
+As a special case, you may provide a `Consumer<T>` argument in addition to other bound parameters.
+The provided consumer is executed once for each row in the result set.
+The static type of parameter T determines the row type.
+
+[source,java]
+----
+@SqlQuery("select id, name from users")
+void forEachUser(Consumer<User> consumer);
+----
```

Comment:
`@SqlQuery` hasn't been properly introduced by this point in the guide. This would probably be a better fit in the next section.